doc/tex/Actors.tex

```diff
@@ -405,7 +405,7 @@ \subsection{Actors from Composable Behaviors \experimental}
 be spawned directly through an actor system by calling
 \lstinline^system.spawn<...>()^, as shown below.
 
-\cppexample[20-52]{composition/calculator_behavior}
+\cppexample[20-45]{composition/calculator_behavior}
```

Comment:
This doesn't match the comment in `calculator_behavior.cpp`. I suspect the comment in the cpp file wasn't updated.

Reply:
I'll make a note. We'll to touch several files for the type ID again, let's check them all at once when doing so.